### ISSUE TITLE
Organize state-of-the-project BoFs

### ISSUE THREAD

**Issue by thewtex** [scipy-conference/conf_2014#20]
Solicit state-of-the-project BoFs like last year. 

  https://github.com/scipy-conference/scipy-conference/blob/master/mailings/bof_announcement.rst

**Comment by mandli** [comment]
So the obvious projects to target here are:
- `numpy`
- `scipy`
- `matplotlib`

and perhaps some of the scikits.  We should start sending emails to either the project mailing lists or approach project leads directly.  Since trying to approach directly last year did not work particularly well, I would suggest we send an email to the dev list (if it exists) in each case and see what kind of response we get.

**Comment by thewtex** [comment]
Yes, we can post directly and ask for spam forgiveness later :-)
